blogs/ecosystems-and-partners/zyphra/README.md

```diff
@@ -0,0 +1,125 @@
+---
+blogpost: true
+date: 10 December 2024
+blog_title: "Zyphra Introduces Frontier Training Kernels for Transformers and SSMs on AMD Instinct MI300X Accelerators"
```

Comment:
Reverting back to an earlier suggested title, please use:
```suggestion
blog_title: "Zyphra Speeding Up Training on AMD Instinct MI300X Accelerators"
```

Reply:
We prefer the previous title because:
1. We want readers to know we've achieved state of the art ("frontier")
2. We wrote kernels for both transformers and SSMs
3. We understand and resolved the main blocker from ROCm training: backward kernels